content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -44,19 +44,32 @@ for your specific needs. The following built-in configuration profiles are curre
 1. **preview**: the preview profile contains features that are experimental. This is intended to explore new features
                 coming to Istio. Stability, security, and performance are not guaranteed - use at your own risk.
 
+1. **ambient**: the ambient profile is designed to help you get started with [ambient mesh](/docs/ops/ambient).
+
+    {{< warning >}}
+    Ambient is currently in [alpha status](/docs/releases/feature-stages/#feature-phase-definitions).
+
+    Please **do not run ambient in production** and be sure to thoroughly review the [feature phase definitions](/docs/releases/feature-stages/#feature-phase-definitions) before use.
+    In particular, there are known performance, stability, and security issues in the `alpha` release.
+    There are also planned breaking changes, including some that will prevent upgrades.
+    These are all limitations that will be addressed before graduation to `beta`.
+    {{< /warning >}}
```

Comment:
I would put this text in boilerplates and then use that here and update the `ambient\getting-started` to use it as well.

Reply:
Yep - good call. I assumed there had to be something like this -- I was searching for the wrong phrases. Has been fixed in the latest push.